rules/run-llms-locally/rule.md

```diff
@@ -0,0 +1,189 @@
+---
+type: rule
+tips: ""
+title: Do you run LLMs locally for enhanced privacy and control?
+seoDescription: Learn how to run large language models locally using Ollama and Microsoft Foundry Local for enhanced privacy, control, and offline capabilities.
+uri: run-llms-locally
+authors:
+  - title: Brady Stroud
+    url: https://www.ssw.com.au/people/brady-stroud
+related:
```

Comment:
https://www.ssw.com.au/rules/choosing-large-language-models/
Might be good as another related rule